manticore/ethereum.py

```diff
@@ -755,15 +762,18 @@ def solidity_create_contract(self, source_code, owner, balance=0, address=None,
         compile_results = self._compile(source_code)
         init_bytecode = compile_results[2]
 
+        if address is None:
+            address = self.world._new_address()
```

Comment:
let's remove the underscore prefix in `world._new_address` since it's a public func?

Reply:
Ideally it should not be public. This qualifies as tech debt